File: Sistema/API/app/application/Model/ClasificacionModel.py

from .BaseModel import BaseModel
from ..Shared import db
from ..Shared import ma

class Clasificacion(BaseModel):
    __tablename__ = 'clasificacion'
    identificador = db.Column(db.String(128))
    edadMinima = db.Column('edad_minima',db.String(128))
    recomendacion = db.Column(db.String(128))
    definicion = db.Column(db.String(500))

class ClasificacionSchema(ma.SQLAlchemyAutoSchema):
    class Meta:
        model = Clasificacion
        load_instance = True

File: Sistema/FE/app/src/app/entities/clasificacion/clasificacion.model.ts
export interface IClasificacion {
    id?: number;
    nombre?: string;
    identificador?: string;
    edadMinima?: number;
    recomendacion?: string;
    definicion?: string;
}

export class Clasificacion implements IClasificacion {
    constructor(
        public id?: number,
        public nombre?: string,
        public identificador?: string,
        public edadMinima?: number,
        public recomendacion?: string,
        public definicion?: string,
    ) {}
}